etc/apache2/vhosts.d/openqa-common.inc

```diff
@@ -29,6 +42,7 @@ ProxyPreserveHost On
 ProxyPass /robots.txt !
 ProxyPass /favicon.ico !
 ProxyPass /images !
+ProxyPass /image !
```

Comment:
I'm not sure here, but wouldn't this prevent /image requests to be passed to Mojo so then thumb_image route defined lower wouldn't be accesible?